tasks/src/main/java/ru/sberbank/school/task02/calculator/ExtendedCurrencyCalculator.java

```diff
@@ -0,0 +1,108 @@
+package ru.sberbank.school.task02.calculator;
+
+import ru.sberbank.school.task02.ExtendedFxConversionService;
+import ru.sberbank.school.task02.ExternalQuotesService;
+import ru.sberbank.school.task02.util.Beneficiary;
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.Quote;
+import ru.sberbank.school.task02.util.Symbol;
+
+import java.lang.management.OperatingSystemMXBean;
+import java.math.BigDecimal;
+import java.util.*;
+import java.math.RoundingMode;
+
+public class ExtendedCurrencyCalculator extends CurrencyCalculator implements ExtendedFxConversionService {
+    private int rounding_mode;
+    Date currentDate = new Date();
+
+    public ExtendedCurrencyCalculator(ExternalQuotesService externalQuotesService) {
+        super(externalQuotesService);
+    }
+
+    private Optional<BigDecimal> extendedOperation(ClientOperation operation, Beneficiary beneficiary, List<Quote> quotes) {
```

Comment:
Line is longer than 120 characters (found 124).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-2016136204>*